.circleci/config.yml

```diff
@@ -51,7 +55,41 @@ jobs:
       - run: yarn test --full --ci
 
 workflows:
-  version: 2
+  version: 2.0
+
+  build_without_deploy:
+    jobs:
+      - docker/publish:
+          deploy: false
+          image: sourcecred/sourcecred
+          tag: latest
+          filters:
+            branches:
+              ignore: 
+                - master
+          after_build:
+            - run:
+                name: Preview Docker Tag for Build
+                command: |
+                   DOCKER_TAG=$(docker run sourcecred/sourcecred --version | cut -d' ' -f2)
+                   echo "Version for Docker tag is ${DOCKER_TAG}"
```

Comment:
Should we keep --version based tagging, I think the text echoed here should reflect it being a preview.
Suggesting `Version that would be used for Docker tag is ${DOCKER_TAG}`

Reply:
See the title "Preview Docker Tag for Build"

Reply:
It's not obvious here, but it jumps out at you in the interface.

![image](https://user-images.githubusercontent.com/814322/63601727-fe5ec980-c593-11e9-8c95-e8a516c592fb.png)

but if the echo is really bothering you please go ahead and change!

Reply:
Making this change, since I'm making the one above too.